tests/unit/s2n_server_new_session_ticket_test.c

```diff
@@ -302,23 +307,33 @@ int main(int argc, char **argv)
     /* s2n_generate_ticket_lifetime */
     {
         uint32_t min_lifetime = 0;
+        struct s2n_config *config = NULL;
         struct s2n_connection *conn = NULL;
         EXPECT_NOT_NULL(conn = s2n_connection_new(S2N_SERVER));
+        EXPECT_NOT_NULL(config = s2n_config_new());
+
+        EXPECT_OK(s2n_resumption_test_ticket_key_setup(config));
```

Comment:
You shouldn't need to set up a resumption test key for this test. You control the inputs to s2n_generate_ticket_lifetime. Just provide the function with the inputs that you want for testing.